modules/jolt_physics/jolt_physics_server_3d.cpp

```diff
@@ -214,6 +214,24 @@ bool JoltPhysicsServer3D::space_is_active(RID p_space) const {
 	return active_spaces.has(space);
 }
 
+void JoltPhysicsServer3D::space_step(RID p_space, real_t p_delta) {
+	JoltSpace3D *space = space_owner.get_or_null(p_space);
+	ERR_FAIL_NULL(space);
+	ERR_FAIL_COND(space->is_stepping());
+
+	space->step(p_delta);
```

Comment:
```suggestion
	job_system->pre_step();
	space->step(p_delta);
	job_system->post_step();
```

As per @mihe 's suggestion in https://github.com/jrouwe/JoltPhysics/issues/1701#issuecomment-3308793660

This addresses the 'Jolt Physics job system exceeded the maximum number of jobs.' warning